src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+        //                     count++;
+        //                     System.out.println("Added new deadline:\n  " + t + "\nNumber of tasks: " + count);
+        //                     word = br.readLine().strip().split(" ",2);
+        //                 } catch (DateTimeParseException e) {
+        //                     throw new DukeException("Date after /by needs to be in format yyyy-mm-dd");
+        //                 }
+        //             } else if (word[0].equals("event")) {
+        //                 if (word.length == 1 || !word[1].contains("/from") || !word[1].contains("/to") ) {
+        //                     throw new DukeException("Event needs a /from and /to.");
+        //                 }
+        //                 String[] tempWord = word[1].split("/");
+        //                 String[] from = tempWord[1].split(" ",2);
+        //                 String[] to = tempWord[2].split(" ",2);
+        //                 if (from.length == 1 || to.length == 1) {
+        //                     throw new DukeException("/from and /to needs a date/time.");
+        //                 }
+        //                 try {
+        //                     Task t = new Event(tempWord[0].strip(), from[1].strip(), to[1].strip());
+        //                     lst.add(t);
+        //                     count++;
+        //                     System.out.println("Added new event:\n  " + t + "\nNumber of tasks: " + count);
+        //                     word = br.readLine().strip().split(" ",2);
+        //                 } catch (DateTimeParseException e) {
+        //                     throw new DukeException("Date after /from and /to needs to be in format yyyy-mm-dd");
+        //                 }
+        //             } else if (word[0].equals("delete")) {
+        //                 if (word.length == 1) {
+        //                     throw new DukeException("Delete needs a number.");
+        //                 }
+        //                 if (Integer.parseInt(word[1]) > count) {
+        //                     throw new DukeException("Invalid task.");
+        //                 }
+        //                 Task t = lst.remove(Integer.parseInt(word[1]) - 1);
+        //                 count--;
+        //                 System.out.println("Deleted task:\n  " + t + "\nNumber of tasks: " + count);
+        //                 word = br.readLine().strip().split(" ",2);
+        //             } else {
+        //                 throw new DukeException("Sorry I do not understand the command");
+        //             }
+        //         } catch (DukeException e) {
+        //             System.out.println(e.getMessage());
+        //             word = br.readLine().strip().split(" ",2);
+        //         }
+        //     }
+        //     storeData(lst);
+        //     System.out.println("Duke: Goodbye");
+        // } catch (IOException e) {
+        //     System.out.println(e.getMessage());
+        // }
```

Comment:
I suggest you to just delete all these unnecessary commented code. That is mainly the point of a version control system like git which is to undo any changes that you might not like in the future. Commenting them out and leaving them in code makes your code unnecssarily long and confusing.